libs/ha/ha_common_libs_k8s.py

```diff
@@ -1907,3 +1922,55 @@ def form_endpoint_port(pod_obj, pod_list, port_name="rgw-https"):
             pod_ep_dict[pod] = ip_port
 
         return pod_ep_dict
+
+    def create_iam_user_with_header(self, num_users=1, header=None):
```

Comment:
don't we already have a method in rest lib for this and delete which we can use?

Reply:
Existing methods use decorators for csm login, and we wanted to create/delete without login again. (i.e. we wanted to use existing login).

Reply:
> don't we already have a method in rest lib for this and delete which we can use?

All libs have CSM rest login decorator. Checked in all files